### DISCUSSION TITLE
PDM support

### DISCUSSION THREAD

**Discussion by JohnMoutafis** [DavidVujic/python-polylith#135] [Ideas]
Since [PDM](https://pdm-project.org/latest/) seems to have taken off lately and also provides a cleaner integration with `pyproject.toml` (PEP [517](https://peps.python.org/pep-0517/), [621](https://peps.python.org/pep-0621/)), is there any plan about adding Polylithic support for PDM?

**Comment by DavidVujic** [comment]
Hi @JohnMoutafis!

I have a long-term goal to develop a standalone CLI, that isn't tied to Poetry in specific. Recently, I got a similar question about Hatch (https://github.com/DavidVujic/python-polylith/discussions/100). Currently, it is all-in Poetry though. But PDM seems like a very nice tool and I like the fact that it is PEP compliant!

(moving this into the discussions section)

**Reply by JohnMoutafis** [reply]
@DavidVujic That would be very interesting indeed :)
Thank you for the info!

**Comment by DavidVujic** [comment]
Quick update: I'm working on this right now and this is a first step towards supporting `PDM`!
https://github.com/DavidVujic/python-polylith/pull/144

**Reply by JohnMoutafis** [reply]
Good to know @DavidVujic :)

**Comment by DavidVujic** [comment]
I just announced the PDM support 😄 
https://github.com/DavidVujic/python-polylith/discussions/161

**Reply by JohnMoutafis** [reply]
Nice :smiley:
